rules-presets/map-traffic/map-file.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Map Local File
+description: "Creating a transparent mapping or redirect to a DAT file containing the HTTP request while using Fiddler's rules."
+slug: adv_map_local_file
+position: 30
+---
+
+# Map Local Files
+
+Remote mapping of a request location means testing changes or fixes locally and avoiding deploying on an actual remote host. However, testing against a localhost environment is often the more straightforward, quicker, and cheaper solution. Mapping HTTP requests can be done to remote locations and local sources like files, Fiddler's predefined responses, or manual responses (through the Fiddler user interface). 
+
+With Fiddler Everywhere, you can easily create a rule that **maps HTTP requests to a local DAT file** (which contains the modified HTTP response).
+
+## Creating a "Map Local (File)" Rule
+
+Create a "Map Local (File)" rule by setting the following actions through the [Rules Builder]({%slug modify-traffic-get-started%}).
+
+1. Create a matching condition that uses the "When **all these conditions** are met **any number of times**" pattern. 
+
+1. Match by a **URL** that uses a string value to match the desired URL. For example: **example.com**.
+
+1. (Optional) If you need to test a URL that your DNS server cannot resolve, create a **Return CONNECT Tunnel** action.
+
+1. Create a **Return File** action and set the path to the DAT file that contains the modified HTTP request. For example:
```

Comment:
Is this part correct? Is this the sample path or the contents of the sample DAT file? The screenshot shows a file path and not the file contents. I think this line might need additional work.